oneflow/extension/stack/stacktrace.h

```diff
@@ -3757,6 +3742,18 @@ class Printer {
     return os;
   }
 
+  bool is_oneflow_object(const std::string& obj_absolute_filename) {
+    std::string oneflow_obj_filename = "oneflow";
+    auto pos = obj_absolute_filename.find_last_of("/");
+    if (pos != std::string::npos) {
+      return obj_absolute_filename.substr(pos + 1, obj_absolute_filename.size() - pos - 1)
+                 .find("oneflow")
+             != std::string::npos;
+    } else {
+      return obj_absolute_filename.find("oneflow") != std::string::npos;
+    }
```

Comment:
升级 c++17 之后可以用 filesystem 库了，include <filesystem> 之后，
```suggestion
    return std::filesystem::path(str).filename.find("oneflow") != std::string::npos;
```